lib/ansible/cli/arguments/option_helpers.py

```diff
@@ -155,7 +155,7 @@ def _gitinfo():
 def version(prog=None):
     """ return ansible version """
     if prog:
-        result = [" ".join((prog, __version__))]
+        result = ["{0} [core {1}] ".format(prog, __version__)]
```

Comment:
nit: the readability is better when the placeholders are named
```suggestion
        result = ["{name} [core {ver}] ".format(name=prog, ver=__version__)]
```
also, this makes it easier to convert to using f-strings once py2 is dropped.

Reply:
I would skip 'naming' as that is prone to change, just putting the path to the executable should be clear enough differentiation from the package.